src/ConvertInputFormat/main.cpp

```diff
@@ -375,6 +373,15 @@ int main(int argc, const char *argv[])
             "--format",                                                        // Flag token.
             outputTypeValidation);
 
+    opt.add("",                              // Default.
+            0,                               // Required?
+            0,                               // Number of args expected.
+            0,                               // Delimiter if expecting multiple args.
+            "Do not convert HVACTemplate objects.", // Help description.
+            "-n",                            // Flag token.
+            "--noHVACTemplate"               // Flag token.
```

Comment:
New command line option for ConvertInputFormat to disable converting HVACTemplate objects.